src/main/java/view/OutputView.java

```diff
@@ -1,26 +1,73 @@
 package view;
 
-import common.Information;
-import java.util.List;
+import message.PrintMessage;
 import model.Car;
 import model.MoveStatus;
 
-public class OutputView {
-  private static final String DELIMITER = " : ";
+import java.io.BufferedWriter;
+import java.io.IOException;
+import java.io.OutputStreamWriter;
+import java.util.List;
+
+public class OutputView implements AutoCloseable {
+    private static final String DELIMITER = " : ";
+    private final BufferedWriter writer;
 
-  public void printResultComment() {
-    System.out.println(System.lineSeparator() + Information.RESULT_COMMENT.getMessage());
-  }
+    public OutputView() {
+        this.writer = new BufferedWriter(new OutputStreamWriter(System.out));
+    }
 
-  public void printResult(List<Car> cars) {
-    for (Car car : cars) {
-      System.out.println(car.getName() + DELIMITER + MoveStatus.join("", car.getStatuses()));
+    public void printResultComment() {
+        try {
+            writer.write(System.lineSeparator() + PrintMessage.RESULT_COMMENT.getMessage());
+            writer.newLine();
+        } catch (IOException e) {
+            throw new RuntimeException(e);
+        }
     }
-    System.out.println();
-  }
 
-  public void printWinner(List<Car> cars) {
-    String winnersName = String.join(", ", cars.stream().map(Car::getName).toList());
-    System.out.printf((Information.WINNER.getMessage()) + "%n", winnersName);
-  }
+    public void printResult(List<Car> cars) {
+        cars.forEach(this::printCarRacingResult);
+        try {
+            writer.newLine();
+        } catch (IOException e) {
+            throw new RuntimeException(e);
+        }
+    }
+
+    private void printCarRacingResult(Car car) {
+        try {
+            writer.write(car.getName() + DELIMITER + concatenateMoveStatuses(car.getStatuses()));
+            writer.newLine();
+        } catch (IOException e) {
+            throw new RuntimeException(e);
+        }
+    }
+
+    private String concatenateMoveStatuses(List<MoveStatus> moveStatuses) {
+        long forwardStatusCount = moveStatuses.stream().filter(status -> status.equals(MoveStatus.FORWARD)).count();
+        String forwardString = "-";
+        return forwardString.repeat((int) forwardStatusCount);
```

Comment:
```suggestion
    private String concatenateMoveStatuses(List<MoveStatus> moveStatuses) {
        int forwardStatusCount = (int) moveStatuses.stream().filter(status -> status.equals(MoveStatus.FORWARD)).count();
        String forwardString = "-";
        return forwardString.repeat(forwardStatusCount);
```
사용하는 값을 의도한 형태로 최대한 빨리 바꿔주면 나중에 실수할 확률이 줄어들더라구요
예외를 웬만하면 터지는 곳에서 바로 잡아주는 이유와 비슷한 것 같아요